Sources/SwiftProcessing/Core/UIKitElements/Capture.swift

```diff
@@ -0,0 +1,347 @@
+                       UIInterfaceOrientation.landscapeRight : AVCaptureVideoOrientation.landscapeRight,
+                       UIInterfaceOrientation.portrait : AVCaptureVideoOrientation.portrait,
+                       UIInterfaceOrientation.portraitUpsideDown : AVCaptureVideoOrientation.portraitUpsideDown]
+    
+    var orientationWords = ["up" : AVCaptureVideoOrientation.portrait,
+                            "upsidedown" : AVCaptureVideoOrientation.portraitUpsideDown,
+                            "left" : AVCaptureVideoOrientation.landscapeLeft,
+                            "right" : AVCaptureVideoOrientation.landscapeRight]
+    
+    init(_ view: Sketch) {
+        self.previewView = UIView()
+        self.alert = Alert("default", "default")
+        super.init(view, self.previewView!)
+    }
+    
+    open func setResolution(_ resolution: String) {
+        if let captureQuality = self.AVCaptureQuality[resolution] {
+            self.session.sessionPreset = captureQuality
+        } else {
+            print("Wrong Resolution Key Word")
+        }
+    }
+    
+    open func setFrameRate(_ desiredFrameRate: Float64) {
+        guard let range = self.cameraDevice!.formats.first!.videoSupportedFrameRateRanges.first,
+            range.minFrameRate...range.maxFrameRate ~= (desiredFrameRate)
+            else {
+                print("Requested FPS is not supported by the device's activeFormat !")
+                return
+        }
+        
+        do {
+            try self.cameraDevice!.lockForConfiguration()
+            self.cameraDevice!.activeVideoMaxFrameDuration = CMTimeMake(value: 1,timescale: Int32(desiredFrameRate))
+            self.cameraDevice!.unlockForConfiguration()
+        } catch {
+            print("Failure when locking Configuration")
+        }
+    }
+    
+    private var cameraPosition = AVCaptureDevice.Position.front {
+        didSet {
+            switch cameraPosition {
+            case .front: snapshotImageOrientation = .upMirrored
+            case .unspecified, .back: fallthrough
+            @unknown default: snapshotImageOrientation = .up
+            }
+        }
+    }
+    
+    open func getCameraPosition(_ position: String) -> AVCaptureDevice.Position {
+        if let AVPosition = self.AVCapturePositions[position] {
+            return AVPosition
+        } else {
+            return AVCaptureDevice.Position.front
+        }
+    }
+    
+    func setPhoto(_ width: CGFloat? = nil,_ height: CGFloat? = nil,_ x: CGFloat = 0,_ y: CGFloat = 0,_ finished: @escaping () -> Void) {
+        self.capturePhoto { image in
+            self.photo =  Image(image)
+            if width != nil && height != nil {
+                self.photo = self.photo!.get(x,y,width!,height!)
+            }
+            finished()
+        }
+        
+    }
+    
+    open func get(_ width: CGFloat? = nil,_ height: CGFloat? = nil,_ x: CGFloat = 0,_ y: CGFloat = 0) -> Image? {
```

Comment:
last comment :) 

Lets switch this around to match the order of other functions. x, y, width, height